### DISCUSSION TITLE
Rewind when book has been paused

### DISCUSSION THREAD

**Discussion by purple-emily** [LeoKlaus/plappa#36] [Feature Requests]
This feature is present in the app `BookPlayer` and is called `Smart Rewind`

![700CD9A5-E6B2-42C9-A3C2-69E96CA295BB](https://github.com/LeoKlaus/plappa/assets/93710772/bfa0c50a-4abd-4221-b8f5-5e6afcb30db8)

When you pause the book for a period of time, it automatically rewinds it a small amount. I love this feature as it allows me to recap on what was the last thing I listened to.

**Comment by LeoKlaus** [comment]
I just implemented this, it will release with the next update!
